public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -260,6 +226,51 @@ export const DataConnection = (props: { dataSource: string }) => {
     },
   ];
 
+  const conditionalTabs =
+    datasourceDetails.connector !== 'PROMETHEUS'
```

Comment:
Instead of doing the checking of `PROMETHEUS` should we reverse the logic here for checking if it is `S3GLUE`?

Reply:
I suspect that'd be better, not sure if we plan on supporting the current glue options for any other data sources

Reply:
fixed